packages/parallax/src/index.tsx

```diff
@@ -46,40 +48,58 @@ export interface ParallaxLayerProps extends ViewProps {
   offset?: number
   /** Shifts the layer in accordance to its offset, values can be positive or negative */
   speed?: number
+  /** Layer will be sticky between these two offsets, all other props are ignored */
+  sticky?: { from: number; to: number }
 }
 
 export const ParallaxLayer = React.memo(
   React.forwardRef<IParallaxLayer, ParallaxLayerProps>(
-    ({ horizontal, factor = 1, offset = 0, speed = 0, ...rest }, ref) => {
+    (
+      { horizontal, factor = 1, offset = 0, speed = 0, sticky, ...rest },
+      ref
+    ) => {
       // Our parent controls our height and position.
       const parent = useContext<IParallax>(ParentContext)
 
       // This is how we animate.
       const ctrl = useMemoOne(() => {
-        const targetScroll = Math.floor(offset) * parent.space
-        const distance = parent.space * offset + targetScroll * speed
+        let translate
+        if (sticky) {
+          translate = sticky.from * parent.space
```

Comment:
do we need a default for `sticky.from` how does a controller normally handle is if there no `from` defined but a `to` prop?

Reply:
My initial thoughts are: `sticky.from` should default to `0` and if no `sticky.to` is set, the layer should just always be sticky whenever the container is scrolled above `sticky.from`. Ideally `sticky.to` would default to `parent.pages * parent.space`, but then we'd have to include `pages` in `ParentContext`. 

I don't know if the behavior really mirrors `from`/`to` of `Controller` and similar APIs, so maybe it should be changed to something like `start`/`end`?

Reply:
`0` makes sense to me. Maybe `to` could be `1`? or `1` greater than `from`?

Yeah `start` and `end` makes more sense to me I think, is that weird in comparison to the other `parallax` props?

Reply:
Ooh, I like `to = from + 1`! 

As far as the names, I don't think it'd be too weird. Either choice will sound a little off I think because most `from`/`to` props in react-spring are referring to interpolation of some sort and this is about offsets. I only picked `from`/`to` initially for consistency, but I also liked how it sounded out loud, "this layer is sticky `from` [here] `to` [here]."